src/Returns.php

```diff
@@ -0,0 +1,328 @@
+        $response = $this->getArtifact(
+            $endpoint,
+            $options
+        );
+        return $response;
+    }
+
+    /**
+     * Get next open return artifact.
+     *
+     * @param string $from
+     *   The beginning range. YmdHs format, eg. 201808282359.
+     * @param string $to
+     *   The end range, defaults to current time. YmdHs format, eg. 201808282359.
+     * @param array $options
+     *   The options array.
+     *
+     * @return \DOMDocument|\Psr\Http\Message\StreamInterface
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getAllOpenReturns($from, $to = '', array $options = []) {
+        if (empty($to)) {
+            $to = date('YmdHs');
+        }
+        $query_params = "from={$from}&to{$to}";
+
+
+        $response = $this->get(
+            "rs/{$this->config['customer_number']}/{$this->config['customer_number']}/openreturn?{$query_params}",
+            [
+                'Accept' => 'application/vnd.cpc.openreturn+xml',
+                'Content-Type' => 'application/vnd.cpc.openreturn+xml',
+            ],
+            $options
+        );
+        return $response;
+
+    }
+
+    /**
+     * Delete the open return template.
+     *
+     * @param string $endpoint
+     *   The endpoint of the file to delete from the Canada Post server.
+     * @param array $options
+     *   The options array.
+     *
+     * @return \DOMDocument|\Psr\Http\Message\StreamInterface
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function deleteOpenReturnTemplate($endpoint, array $options = []) {
+        $response = $this->delete(
+            $endpoint,
+            [
+                'Accept' => 'application/vnd.cpc.openreturn+xml',
+                'Content-Type' => 'application/vnd.cpc.openreturn+xml',
+                'Accept-Language' => 'en-CA'
+            ],
+            $options
+        );
+        return $response;
+    }
+
+    /**
+     * Helper function to build the content array.
+     *
+     * @param array $returner
+     *   The returner info.
+     * @param array $receiver
+     *   The receiver info.
+     * @param array $parcel
+     *   The parcel characteristics.
+     * @param array $options
+     *   The options array.
+     *
+     * @return array
+     *   The content array.
+     */
+    protected function buildReturnsArray(
+        array $returner,
+        array $receiver,
+        array $parcel,
+        array $options = []
+    ) {
+        $this->formatPostalCode($returner);
+        $this->formatPostalCode($receiver);
+        $return_info = [
+            'service-code' => $parcel['service_code'],
+            'returner' => $returner,
+            'receiver' => $receiver,
+            'parcel-characteristics' => $parcel,
+        ];
+
+        if (!empty($options['option_codes'])) {
+            $return_info['options']['option'] = $this->parseOptionCodes($options);
+        }
+
+        return $return_info;
+    }
```

Comment:
**CHANGE REQUEST**

This function (`buildReturnsArray`) does not seem to be called anywhere in the code?